webapp/src/workspace.ts

```diff
@@ -927,29 +928,48 @@ export async function importGithubAsync(id: string): Promise<Header> {
 
     let sha = ""
     let isEmpty = false
+    let forceTemplateFiles = false;
     try {
         sha = await pxt.github.getRefAsync(parsed.fullName, parsed.tag)
+        // if the repo does not have a pxt.json file, treat as empty 
+        // (must be done before)
+        const commit = await pxt.github.getCommitAsync(parsed.fullName, sha)
+        if (!commit.tree.tree.find(f => f.path == pxt.CONFIG_NAME)) {
+            pxt.log(`github: detected import non-makecode project`)
+            isEmpty = true; // needs initialization
+            forceTemplateFiles = false;
+            // ask user before modifying project
+            const r = await core.confirmAsync({
```

Comment:
Does clicking confirm here also delete non-pxt files?

Reply:
I've added ``Your existing files will not be modified.``. It leaves all the other files alone.